demos/tests/cases.py

```diff
@@ -358,6 +423,43 @@ def single_option_cases(key, *args):
             ModelArg('instance-segmentation-security-1025')),
     )),
 
+    PythonDemo(subdirectory='machine_translation_demo', device_keys=[], test_cases=combine_cases(
+       [
+           TestCase(options={
+               '-m': ModelArg("machine-translation-nar-en-ru-0001"),
+               '--tokenizer-src': OmzDirArg('models/intel/machine-translation-nar-en-ru-0001/tokenizer_src'),
+               '--tokenizer-tgt': OmzDirArg('models/intel/machine-translation-nar-en-ru-0001/tokenizer_tgt'),
+               '--output-name': 'pred',
+               '-i': [
+                   "A robot may not injure a human being or, through inaction, allow a human being to come to harm.",
+                   "A robot must obey orders given it by human beings except where such orders would conflict with the "
+                   "First Law.",
+                   "A robot must protect its own existence as long as such protection does not conflict with the First"
+                   " or Second Law."
```

Comment:
It's a nice reference, but it is from a copyrighted book. I think we should play it safe and use original or public domain text.

Reply:
@yssemaev can't we quote text from art books? might be enough to put reference to the author/book somewhere?

Reply:
I don't think we need to obsess much over this. Just use a public domain book, there's loads of them.

Reply:
Done